src/config.rs

```diff
@@ -14,6 +15,9 @@ pub struct AmConfig {
     #[serde(rename = "endpoint")]
     pub endpoints: Option<Vec<Endpoint>>,
 
+    /// Prometheus endpoint which should get passed to Dora when invoking `am explore`
```

Comment:
Do not call this Dora

Reply:
I don't think we should add the `prometheus_endpoint` to the am.config tbh. The `am start` for example doesn't use this, and might be surprising that because of a change of another user you'd somehow open a different explorer url.

I think it makes sense to have a per user configuration which can contain multiple prometheus endpoints, if the user didn't use a override, then we can show a picker.